src/plugins/section/commands.js

```diff
@@ -8,8 +8,9 @@ import { Block, Range, Text } from 'slate'
  * Create a new section.
  *
  * @param {Slate~Change} change
+ * @param {string} titleText
  */
-export function insertSection(change) {
+export function insertSection(change, titleText = "") {
```

Comment:
I'd prefer the decision what to do if title text isn't specified was left to Slate.